### DISCUSSION TITLE
Replicated join 的优化使用有前提条件吗？

### DISCUSSION THREAD

**Discussion by josonle** [StarRocks/starrocks#3926] [Announcements]
![image](https://user-images.githubusercontent.com/23043019/157158911-c1fb81e0-9040-4cc4-9c01-628f819d7814.png)

如图，看去年有replicated join相关pr的提交，starrocks应该已经支持该特性。想了解表关联时是否使用该特性需要提前设置什么吗？比如说副本数设置成和节点数一致等等

**Comment by chaoyli** [comment]
@josonle Currently, the replication_join is not used because the performance is not outperformed than broadcast. Yes, it’s to replace broadcast join. The the benchmark result it not convincing. Need more refactor to optimize the replicated_join.
